react/SearchResult.js

```diff
@@ -109,16 +138,12 @@ class SearchResult extends Component {
   }
 
   countSelecteds(selecteds) {
-    let count = 0
-    for (const key in selecteds) {
-      if (selecteds[key]) {
-        count += selecteds[key].length
-      }
-    }
-    return count
+    // return 0
+    if (!selecteds) return 0
+    return reduce(concat, [])(values(selecteds)).length
```

Comment:
Really better, try this final touch:

  `countSelecteds = selecteds => return selecteds ? reduce(...).length : 0`